### PR TITLE
update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by petermakowski**
## Done

- replace `./run` with `dotrun` in `PULL_REQUEST_TEMPLATE.md` instructions

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- [Add additional steps]

## Details

[List of links to issues/bugs and cards if needed]

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ## QA
 
 - Pull code
-- Run `./run clean && ./run serve`
+- Run `dotrun clean && dotrun serve`
```
Comment (hatched) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: fwiw I would always just run this locally with yarn as there are no global dependencies. `yarn && yarn start`

Comment (petermakowski) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ohh that's a good point. Thanks @hatched 🎉

**Review by hatched — APPROVED** [review]

**Comment by petermakowski** [comment]
@anthonydillon replaced `yarn` with `dotrun`

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**